stack.yaml

```diff
@@ -1,2 +1,5 @@
 resolver: lts-8.6
 packages: ['.']
+nix:
+  enable: false
+  packages: [zlib]
```

Comment:
Can you explain to me why you added this?
I'm guessing this is for Linux stuff?